bench/macro/lsm-tree-bench-bloomfilter.hs

```diff
@@ -13,7 +13,7 @@ import           Data.BloomFilter (Bloom)
 import qualified Data.BloomFilter as Bloom
 import qualified Data.BloomFilter.Hash as Bloom
 import qualified Data.BloomFilter.Mutable as MBloom
-import           Data.List (foldl')
+import qualified Data.List as List
```

Comment:
This is actually "wrong". `import qualified Data.Foldable as Foldable`, as we use it on vector, not the list.

Reply:
(alternatively, `Vector.foldl'`)

Reply:
Really, I'm surprised it compiled, the tests ran, and the benchmarks ran if it was wrong. Maybe I misunderstand what you mean by "wrong" here? Regardless of any misunderstanding, I can easily swap the `foldl'` references to the polymorphic definition from `Data.Foldable`.

Reply:
`Data.List` exports the `Foldable`-polymorphic versions, so it compiles for vectors as well. It's just a weird/misleading place to get them from if you're not using them for lists.

Reply:
Ah, got it. That makes sense. I recall the GHC warning that this will change in an upcoming version; i.e. that `Data.List.foldl'` will be monomorphic to lists.

Reply:
The requested changes have been made.